src/components/TimeseriesDataExport/TimeseriesDataExport.tsx

```diff
@@ -135,13 +135,13 @@ const TimeseriesDataExportFC: FC<TimeseriesDataExportFormProps> = (
     const { start = 0, end = 0 } = await getLimits(request);
     const numericGranularity = getGranularityInMS(granularity);
     const msPerRequest =
-      (numericGranularity * CELL_LIMIT) / timeseriesIds.length;
+      (numericGranularity * CELL_LIMIT) / seriesNumber;
```

Comment:
Is that because a series may contain multiple timeseries?

Reply:
SeriesNumber depends on both `timeseriesIds` and `timeseriesExternalIds`, and when I've been testing I've only passed `timeseriesExternalIds` for the charts, so limit became wrongly set